control-plane/p4RuntimeArchStandard.h

```diff
@@ -23,13 +23,201 @@ limitations under the License.
 
 #include "p4RuntimeArchHandler.h"
 
+namespace p4configv1 = ::p4::config::v1;
+
+using ::P4::ControlPlaneAPI::Helpers::getExternInstanceFromProperty;
+using ::P4::ControlPlaneAPI::Helpers::setPreamble;
+
 namespace P4 {
 
 /** \addtogroup control_plane
  *  @{
  */
 namespace ControlPlaneAPI {
 
+namespace Standard {
```

Comment:
I am assuming this has been moved from the cpp file. Github diff makes it difficult to check.

Reply:
Yes, this code needed to be in the header file to allow extending the class.